app/build.gradle

```diff
@@ -44,6 +44,9 @@ dependencies {
     implementation "io.reactivex.rxjava3:rxjava:$rootProject.rx_java_version"
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
 
+    //Retrofit to RxJava CallAdapter Factory
+    implementation 'com.squareup.retrofit2:adapter-rxjava3:2.9.0'
```

Comment:
also here you can directly use `$rootProject.retrofit_version` to maintain uniformity in the codebase

Reply:
Done